### PR TITLE
Changing relative paths for parsers Unroll gtests

### PR DESCRIPTION
**Pull request by VolodymyrPeschanenkoLitSoft**
These changes allow running gtestp4c from different folders.

### CODE REVIEW

File: test/gtest/parser_unroll.cpp
Comment (MichalKekely) [inline test/gtest/parser_unroll.cpp:238]: `relPath` at this point points to `<p4c>/`. This means that the `includeDir` is just fine, but `options.file` points for example to `<p4c>/parser-unroll-test1.p4` not to the `<p4c>/testdata/p4_16_samples/parser-unroll-test1.p4`. Or even worse to `<p4c>/../p4_14_samples/switch_20160512/switch.p4` in case of the `switch_20160512` test. I am guessing the full paths within the p4c root directory need to be added for all of the `loadExample()` calls.

Comment (VolodymyrPeschanenkoLitSoft) [reply test/gtest/parser_unroll.cpp:238]: Yes, you are right, thanks...

File: test/gtest/parser_unroll.cpp
Comment (MichalKekely) [inline test/gtest/parser_unroll.cpp:244]:
```suggestion
    options.file += "testdata/p4_16_samples/";
```

Comment (VolodymyrPeschanenkoLitSoft) [reply test/gtest/parser_unroll.cpp]: Looks strange, how it passes locally.

**Comment by MichalKekely** [comment]
@VolodymyrPeschanenkoLitSoft This looks good now. Thanks for the fix.

**Comment by MichalKekely** [comment]
@VolodymyrPeschanenkoLitSoft Could you merge this? Maybe after approval from someone else than just me :-)

**Comment by VolodymyrPeschanenkoLitSoft** [comment]
> @VolodymyrPeschanenkoLitSoft Could you merge this? Maybe after approval from someone else than just me :-)

Unfortunately, I haven't written the access for merging. @mbudiu-vmw could, you, please, make a review of these changes for us?

**Review by mbudiu-vmw — APPROVED** [review]

**Pull request merged**
